### PR TITLE
PMM-10640 Added new PSMDB image for testing

### PR DESCRIPTION
**Pull request by percona-csalguero**

### CODE REVIEW

**Review by BupycHuk — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

**Review by denisok — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

**Review by denisok — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

File: sources/operator.1.10.0.psmdb-operator.json [hunk 1]
```diff
@@ -5,6 +5,12 @@
       "product": "psmdb-operator",
       "matrix": {
         "mongod": {
+          "5.0.7-6": {
```
Comment (tplavcic) [inline sources/operator.1.10.0.psmdb-operator.json:8]: If I'm not mistaking this is dangerous! We will add a new version for the old operator version and anybody having automatic upgrades enabled (and automatic upgrades in the old version were enabled by default) will get automatic upgrade.

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: sources/operator.1.10.0.psmdb-operator.json (hunk 1 above)
Comment (hors) [reply sources/operator.1.10.0.psmdb-operator.json:8]: one more thing It was not tested with 1.10.0 operator at all

**Pull request closed without merging**